content/rc/security/private-service-connect.md

```diff
@@ -132,6 +132,10 @@ Once your Private Service Connect endpoint is active, you can connect your appli
 
 1. Copy the endpoint and use it in your application to connect to your database.
 
+{{< note >}}
+Once your Private Service Connect endpoint is active, you must connect to the database from the Private Service Connect endpoint. If you previously used the public or private endpoint to connect to your database, you must migrate any connections to the Private Service Connect endpoint.
```

Comment:
@cmilesb do you think we should also mention that customers moving away from PSC should change their endpoint to the public or private endpoint or do you think the above implies it?

Reply:
Other than that looks good :) Thanks!

Reply:
Just added something under deactivate in [eb06970](https://github.com/RedisLabs/redislabs-docs/pull/3195/commits/eb06970dcb6c3cd955e292dd558c4b1c5c1ef529)